app/models/comment.rb

```diff
@@ -132,18 +132,22 @@ def notify_users(uids, current_user)
   # email all users in this thread
   # plus all who've starred it
   def notify(current_user)
-    if parent.uid != current_user.uid && !UserTag.exists?(parent.uid, 'notify-comment-direct:false')
-      CommentMailer.notify_note_author(parent.author, self).deliver_now
-    end
+    if current_user.first_time_commenter
```

Comment:
Oh, okay! I put it like this because you told me that the `first_time_commenter` function would cover both methods. I'm gonna add the line right now! 

Thanks!